### DISCUSSION TITLE
transfer-02-consumer-pull proxyPath is encoded - Can this be turned off?

### DISCUSSION THREAD

**Discussion by oliverbusch** [eclipse-edc/Samples#205] [Q&A] [answered]
When connecting to a backend service, there is a problem if the ProxyPath is not to be encoded.

Here is an example:

The connected backend service requires the following parameter to provide the data:

`smart.data%2Fdemo%2Frepo%2F1%2F1%2F454576463545648365874`

The transfer is done with the following command 
```
curl --location --request GET 'http://localhost:19191/api/public/1/smart.data%2Fdemo%2Frepo%2F1%2F1%2F454576463545648365874' \
--header 'Authorization: 1234' \
--header 'X-Api key: password'
```

In the DataPlane log console, you can see that the URL is encoded

`DEBUG 2024-02-13T12:34:06.879157137 Executing HTTP request: https://backend/repository/smart.data/demo/repo/1/1/454576463545648365874`

But the backend expects the following url
`https://backend/repository/smart.data%2Fdemo%2Frepo%2F1%2F1%2F454576463545648365874`

Is there a way to disable the decoding of the proxy path? Or is there another way to pass the ProxyPath decoded?

**Comment by ndr-brt** [comment, marked as answer]
hi, currently it's not possible ([currently the path is added using the okhttp `addPathSegments`, that encodes it](https://github.com/eclipse-edc/Connector/blob/132442934cdcf291ad5e7478bb45bbf6f38f847f/extensions/data-plane/data-plane-http/src/main/java/org/eclipse/edc/connector/dataplane/http/params/HttpRequestFactory.java#L102)), but I guess it is a feature that could be implemented. Could you please open an issue in the https://github.com/eclipse-edc/Connector repository?

**Reply by oliverbusch** [reply]
Feature Request created https://github.com/eclipse-edc/Connector/issues/3862
